doc/source/user_guide/groupby.rst

```diff
@@ -476,55 +476,92 @@ Or for an object grouped on multiple columns:
 -----------
 
-Once the GroupBy object has been created, several methods are available to
-perform a computation on the grouped data. These operations are similar to the
-:ref:`aggregating API <basics.aggregate>`, :ref:`window API <window.overview>`,
-and :ref:`resample API <timeseries.aggregate>`.
-
-An obvious one is aggregation via the
-:meth:`~pandas.core.groupby.DataFrameGroupBy.aggregate` or equivalently
-:meth:`~pandas.core.groupby.DataFrameGroupBy.agg` method:
+An aggregation is a GroupBy operation that reduces the dimension of the grouping
+object. The result of an aggregation is, or at least is treated as,
+a scalar value for each column in a group. For example, producing the sum of each
+column in a group of values.
 
 .. ipython:: python
 
-   grouped = df.groupby("A")
-   grouped[["C", "D"]].aggregate(np.sum)
-
-   grouped = df.groupby(["A", "B"])
-   grouped.aggregate(np.sum)
+   animals = pd.DataFrame(
+       {
+           "kind": ["cat", "dog", "cat", "dog"],
+           "height": [9.1, 6.0, 9.5, 34.0],
+           "weight": [7.9, 7.5, 9.9, 198.0],
+       }
+   )
+   animals
+   animals.groupby("kind").sum()
 
-As you can see, the result of the aggregation will have the group names as the
-new index. In the case of multiple keys, the result is a
-:ref:`MultiIndex <advanced.hierarchical>` by default, though this can be
-changed by using the ``as_index`` option:
+In the result, the keys of the groups appear in the index by default. They can be
+instead included in the columns by passing ``as_index=False``.
 
 .. ipython:: python
 
-   grouped = df.groupby(["A", "B"], as_index=False)
-   grouped.aggregate(np.sum)
+   animals.groupby("kind", as_index=False).sum()
 
-   df.groupby("A", as_index=False)[["C", "D"]].sum()
+.. _groupby.aggregate.builtin:
 
-Note that you could use the ``reset_index`` DataFrame function to achieve the
-same result as the column names are stored in the resulting ``MultiIndex``:
+Built-in aggregation methods
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. ipython:: python
+Many common aggregations are built-in to GroupBy objects as methods. Of the methods
+listed below, those with a ``*`` do *not* have a Cython-optimized implementation.
 
-   df.groupby(["A", "B"]).sum().reset_index()
+.. csv-table::
+    :header: "Method", "Description"
+    :widths: 20, 80
+    :delim: ;
+
+        :meth:`~.DataFrameGroupBy.any`;Compute whether any of the values in the groups are truthy
+        :meth:`~.DataFrameGroupBy.all`;Compute whether all of the values in the groups are truthy
+        :meth:`~.DataFrameGroupBy.count`;Compute the number of non-NA values in the groups
+        :meth:`~.DataFrameGroupBy.cov` * ;Compute the covariance of the groups
+        :meth:`~.DataFrameGroupBy.first`;Compute the first occurring value in each group
+        :meth:`~.DataFrameGroupBy.idxmax` *;Compute the index of the maximum value in each group
+        :meth:`~.DataFrameGroupBy.idxmin` *;Compute the index of the minimum value in each group
```

Comment:
This looks weird, does this render properly?

Reply:
![image](https://user-images.githubusercontent.com/45562402/223600429-4c3da473-5b1f-4d96-9e82-46d642f37fb0.png)